api/filter_outputs.go

```diff
@@ -1,14 +1,63 @@
 package api
 
 import (
+	"fmt"
 	"net/http"
+	"strings"
 
 	"github.com/ONSdigital/dp-cantabular-filter-flex-api/model"
 	"github.com/ONSdigital/log.go/v2/log"
 	"github.com/go-chi/chi/v5"
 	"github.com/pkg/errors"
 )
 
+func (api *API) getFilterOutput(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	fID := chi.URLParam(r, "filter-output-id")
+
+	var filterOutput *model.FilterOutput
+
+	filterOutput, err := api.store.GetFilterOutput(ctx, fID)
+	if err != nil {
+
```

Comment:
No need to leave blank lines at the beginning of code blocks. In the long run it's more legible to keep consistent.

Reply:
resolved, and noted